src/probnum/problems/zoo/filtsmooth/_ssm_examples.py

```diff
@@ -0,0 +1,328 @@
+import numpy as np
+
+from probnum import diffeq, filtsmooth, problems, randvars, statespace
+
+__all__ = ["car_tracking", "logistic_ode", "ornstein_uhlenbeck", "pendulum"]
+
+
+def car_tracking(delta_t=0.2, measvar=0.5, t_max=20.0):
+    r"""State space model for a simple car-tracking scenario.
+
+    A linear, time-invariant Gaussian state space model for car-tracking, based on
+    Example 3.6 in Särkkä, 2013. [1]_
+    Let :math:`X = (\dot{x}_1, \dot{x}_2, \ddot{x}_1, \ddot{x}_2)`. Then the state
+    space model has the following discretized formulation
+
+    .. math::
+
+        X(t_{n}) &=
+        \begin{pmatrix}
+          0 & 0 & 1 & 0 \\
+          0 & 0 & 0 & 1 \\
+          0 & 0 & 0 & 0 \\
+          0 & 0 & 0 & 0
+        \end{pmatrix} X(t_{n-1})
+        +
+        q(t_n) \\
+        y_{n} &=
+        \begin{pmatrix}
+          1 & 0 & 0 & 0 \\
+          0 & 1 & 0 & 0 \\
+        \end{pmatrix} X(t_{n})
+        + r_n
+
+    where :math:`q(t_n) \sim \mathcal{N}(0, Q)` and :math:`r_n \sim \mathcal{N}(0, R)`
+    for process noise covariance matrix :math:`Q` and measurement noise covariance
+    matrix :math:`R`.
+
+    Parameters
+    ----------
+    delta_t : float
+        The step size of the discretized time grid on which the model is considered.
+    measvar : float
+        The marginal measurement variance.
+    t_max : float
+        The time limit of the grid.
+
+    Returns
+    -------
+    dynmod
+        The `DiscreteLTIGaussian` dynamics model.
+    measmod
+        The `DiscreteLTIGaussian` measurement model.
+    initrv
+        A `Normal` random variable for the Gaussian initial conditions.
+    regression_problem
+        A `RegressionProblem` with time points and noisy observations.
+
+    References
+    ----------
+    .. [1] Särkkä, Simo. Bayesian Filtering and Smoothing. Cambridge University Press,
+        2013.
+
+    """
+
+    dynamat = np.eye(4) + delta_t * np.diag(np.ones(2), 2)
+    dynadiff = (
+        np.diag(np.array([delta_t ** 3 / 3, delta_t ** 3 / 3, delta_t, delta_t]))
+        + np.diag(np.array([delta_t ** 2 / 2, delta_t ** 2 / 2]), 2)
+        + np.diag(np.array([delta_t ** 2 / 2, delta_t ** 2 / 2]), -2)
+    )
```

Comment:
would be great if these could be replaced by an `IBM` object. Then, we could also expose `ordint` (with a default set to what is happening here)